### PR TITLE
ENH: corrected offset for each blade

### PR DESCRIPTION
**Pull request by ghalym**
ENH: Encoder offset positions updates required for center and gap calculations.

### CODE REVIEW

File: plc-kfe-motion/kfe_motion/POUs/PRG_SL1K0_POWER.TcPOU [hunk 1]
```diff
@@ -101,5 +101,9 @@ END_IF
 
 ]]></ST>
     </Implementation>
+    <LineIds Name="PRG_SL1K0_POWER">
+      <LineId Id="3" Count="31" />
+      <LineId Id="2" Count="0" />
+    </LineIds>
```
Comment (ZLLentz) [inline plc-kfe-motion/kfe_motion/POUs/PRG_SL1K0_POWER.TcPOU:107]: Obligatory re-link to https://confluence.slac.stanford.edu/display/PCDS/TwinCAT+3+Git+Setup+and+Best+Practices#TwinCAT3GitSetupandBestPractices-TwinCATLineIDSettings:LineIDs

**Review by ZLLentz — APPROVED** [review]
Ready to merge?

File: plc-kfe-motion/kfe_motion/POUs/PRG_SL1K0_POWER.TcPOU (hunk 1 above)
Comment (ghalym) [reply plc-kfe-motion/kfe_motion/POUs/PRG_SL1K0_POWER.TcPOU:107]: Sorry, I apparently need to set it everytime I open the project.

**Comment by ghalym** [comment]
Yes, Thank you.

File: plc-kfe-motion/kfe_motion/POUs/PRG_SL1K0_POWER.TcPOU (hunk 1 above)
Comment (ZLLentz) [reply plc-kfe-motion/kfe_motion/POUs/PRG_SL1K0_POWER.TcPOU:107]: That seems a bit ridiculous... something must be wrong

**Pull request merged**